.github/docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# SPDX-FileCopyrightText: 2022 Google LLC
+#
+# SPDX-License-Identifier: Apache-2.0
+FROM ubuntu:22.04
+
+RUN \
+  apt-get update && \
+  apt-get install -y curl xz-utils sudo git && \
+  rm -rf /var/lib/apt/lists/*
+
+RUN \
+  groupadd -r nixbld && \
+  for n in $(seq 1 10); do \
```

Comment:
Why do we need 10 users for the docker? I guess its for different build jobs, but is there some reason that they cannot be executed by the same user instead?

Reply:
Good question. A forum thread suggested this, I'm not sure if the `10` carries any significance. I'll experiment.

Reply:
The documentation says:

> This creates 10 build users. There can never be more concurrent builds than the number of build users, so you may want to increase this if you expect to do many builds at the same time.

So while 10 is a bit arbitrary, I doubt we'll need more than this (and it doesn't hurt to have too many). Agreed?